sdk/core/Azure.Core.TestFramework/src/Generated/TestProxyRestClient.cs

```diff
@@ -265,9 +269,10 @@ internal HttpMessage CreateStopRecordRequest(string xRecordingId, IDictionary<st
         /// <summary> Stop recording for a test. </summary>
         /// <param name="xRecordingId"> The recording ID. </param>
         /// <param name="variables"> Variables for the recording. </param>
+        /// <param name="xRecordingSkip"> Set to request-response to skip recording this session. </param>
         /// <param name="cancellationToken"> The cancellation token to use. </param>
         /// <exception cref="ArgumentNullException"> <paramref name="xRecordingId"/> or <paramref name="variables"/> is null. </exception>
-        public async Task<Response> StopRecordAsync(string xRecordingId, IDictionary<string, string> variables, CancellationToken cancellationToken = default)
+        public async Task<Response> StopRecordAsync(string xRecordingId, IDictionary<string, string> variables, string xRecordingSkip = null, CancellationToken cancellationToken = default)
```

Comment:
Should `xRecordingSkip` be a bool?

Reply:
Test Proxy expects "request-response" as the value here.